timescaledb/how-to-guides/continuous-aggregates/continuous-aggregates-on-continuous-aggregates.md

```diff
@@ -0,0 +1,138 @@
+hourly aggregate. This is more efficient than creating the daily aggregate on
+top of the original hypertable, because you can reuse the calculations from the
+hourly aggregate.
+
+This feature is available in version 2.9 and above of TimescaleDB.
+
+## Create a continuous aggregate on top of another continuous aggregate
+
+Creating a continuous aggregate on top of another continuous aggregate works the
+same way as creating it on top of a hypertable. In your query, select from a
+continuous aggregate rather than from the hypertable, and use the time-bucketed
+column from the existing continuous aggregate as your time column.
+
+For more information, see the instructions for [creating a continuous
+aggregate][create-cagg].
+
+## Use real-time aggregation with stacked continuous aggregates
+
+By default, all continuous aggregates use real-time aggregation. That means they
+always return up-to-date data in response to queries. They accomplish this by
+joining the materialized data in the continuous aggregate with unmaterialized
+raw data from the source table or view.
+
+When continuous aggregates are stacked, each continuous aggregate is only aware
+of the layer immediately below. The joining of unmaterialized data happens
+recursively until it reaches the bottom layer, giving you access to recent data
+down to that layer.
+
+If you keep all continuous aggregates in the stack as real-time aggregates, the
+bottom layer is the source hypertable. That means every continuous aggregate in
+the stack has access to all recent data.
+
+If there is a non-real-time continuous aggregate somewhere in the stack, the recursive
+joining stops at that non-real-time continuous aggregate. Higher-level continuous
+aggregates don't receive any unmaterialized data from lower levels.
+
+For example, say you have the following continuous aggregates:
+
+*   A real-time hourly continuous aggregate on the source hypertable
+*   A real-time daily continuous aggregate on the hourly continuous aggregate
+*   A non-real-time, or materialized-only, monthly continuous aggregate on the
+    daily continuous aggregate
+*   A real-time yearly continuous aggregate on the monthly continuous aggregate
+
+Queries on the hourly and daily continuous aggregates include real-time,
+non-materialized data from the source hypertable. Queries on the monthly
+continuous aggregate only return already-materialized data. Queries on the
+yearly continuous aggregate return materialized data from the yearly continuous
+aggregate itself, plus more recent data from the monthly continuous aggregate.
+However, the data is limited to what is already materialized in the monthly
+continuous aggregate, and doesn't get even more recent data from the source
+hypertable. This happens because the materialized-only continuous aggregate
+provides a stopping point, and the yearly continuous aggregate is unaware of any
+layers beyond that stopping point. This aligns with how stacked views work in
```

Comment:
```suggestion
layers beyond that stopping point. This is similar to how stacked views work in
```

Reply:
Might also want to point to PG docs here?

Reply:
@fabriziomello I got this from the design discussion in the original issue, do you know if it's discussed in the PostgreSQL docs? I tried to search for it but got nothing

Reply:
@charislam here we have an example: https://www.postgresql.org/docs/14/rules-views.html#RULES-SELECT